.circleci/test-deploy.yml

```diff
@@ -165,7 +165,20 @@ jobs:
       - matlab/run-tests:
           test-results-junit: test-results/matlab/filterdtagresult.xml
           select-by-tag: FILTERED
-
+      - matlab/run-tests:
+          select-by-name: $(circleci tests glob 'tests/**/*.m' | circleci tests split | awk -F'[\\\\/.]' '{print $(NF-1) "/*"}')
```

Comment:
Any specific reason why we are running all run-tests and later verifying the log ? can we verify the logs for each run-test?

Reply:
Sure, I have added the verification step after each run-test